draft-ietf-lamps-cms-ml-dsa.md

```diff
@@ -153,19 +141,23 @@ id-ml-dsa-87 OBJECT IDENTIFIER ::= { sigAlgs 19 }
 
 # ML-DSA Key Encoding
 
-{{Section 4 of !I-D.ietf-lamps-dilithium-certificates}} describes the format of ML-DSA public keys when encoded as a part of the SubjectPublicKeyInfo type.
-The signed-data content type as described in {{RFC5652}} does not encode public keys directly, but CMS content types defined by other documents do.
-For example, {{?RFC5272}} describes Certificate Management over CMS, and its PKIData content utilises the SubjectPublicKeyInfo type to encode public keys for certificate requests.
-When the SubjectPublicKeyInfo type is used in CMS, ML-DSA public keys MUST be encoded as described in {{!I-D.ietf-lamps-dilithium-certificates}}
+{{!RFC5280}} defines the SubjectPublicKeyInfo ASN.1 type.
```

Comment:
ietf-lamps-dilithium-certificates refers to use for public key encoding and ANS.1, so I added more text.